setup.cfg

```diff
@@ -18,14 +18,29 @@ version = 0.6
 [options]
 python_requires = >=3.5
 install_requires =
-    numpy==1.13.3; python_version=='3.5' and platform_system!='AIX'
-    numpy==1.13.3; python_version=='3.6' and platform_system!='AIX'
-    numpy==1.14.5; python_version=='3.7' and platform_system!='AIX'
+    # NOTE: platform_system 'AIX' and platform_machine 'aarch64' are mutually
+    #       exclusive so they are specified by themself when setting their
+    #       numpy version.
+
+    # platform_system=='AIX' requires numpy 1.16.0 (needed for Python < 3.8)
     numpy==1.16.0; python_version=='3.5' and platform_system=='AIX'
     numpy==1.16.0; python_version=='3.6' and platform_system=='AIX'
     numpy==1.16.0; python_version=='3.7' and platform_system=='AIX'
-    numpy==1.17.3; python_version=='3.8'
+
+    # platform_machine=='aarch64' requires numpy 1.19.0 (needed for Python < 3.9)
```

Comment:
The first part isn't really true, and the "needed for Python < 3.9" is unnecessary and will go out of date. Can you change to:
```
# numpy 1.19 was the first minor release to provide aarch64 wheels
```

Reply:
And same for the AIX comment above, can you change that to
```
# AIX requires fixes contained in numpy 1.16 
```

Reply:
Adjusting, but I was mainly trying to call out why it stops at a specific Python version and to help future contributors think about that when suggesting changes to this file.